package.json

```diff
@@ -12,8 +12,8 @@
         "build:js": "terser -c -m -o _site/js/cc.min.js node_modules/vanilla-cookieconsent/dist/cookieconsent.js src/js/cookies.js",
         "build": "cross-env NODE_ENV=production npm-run-all clean build:js --parallel prod:*",
         "clean:dev": "cross-env npx del-cli '_site/!(img)' 'src/docs/**/*.md' && cross-env npx mkdirp '_site/js'",
```

Comment:
This is now redundant, it was originally added for the exclusion of the img dir, they can both just use `clean` now

Reply:
Not quite true, we need to clean the docs directly locally: https://github.com/FlowFuse/website/pull/969